data/pirate missions smuggling intro.txt

```diff
@@ -0,0 +1,1174 @@
+			label hook
+			`	She looks saddened. "Well, we'll keep asking you, but if you aren't there when you're needed too often, we'll have to report you to the Republic for breaking contract and collect our 500,000 again. It should go without saying, but abandoning deliveries and missing deadlines is a big no-no. But you don't look like the type of person who would do that! So what do you say?"`
+				goto questions
+			label when
+			`	Alexandra looks at her datapad and taps the screen rapidly. "I'm guessing we'll have something for you within a couple weeks. Of course, we'll only contact you when you're the closest courier, but you can expect to be called up about 10 days after each run."`
+				goto questions
+			label somewhere
+			`	"Not really. Most of our work is in Republic space, and we would prefer it if you tried to stay close. But if you want to catch a few extra jobs here or there, feel free!"`
+				goto questions
+			
+			label yes
+			`	Alexandra smiles. "Great! I'll be in touch, <first>. In the meantime, enjoy your new-found freedom!"`
+				goto meetup7
+			label angel
+			`	Alexandra laughs. "Oh, this is going to be fun. I'm looking forward to working with you, <first>. And, no more debt! Enjoy your freedom!"`
+				goto meetup7
+			
+			label meetup7
+			`	When you make it back to your ship, you wonder if what Alexandra said is too good to be true; but sure enough, 500,000 credits have been transferred to your account; more money than you thought you'd see in a lifetime, and more than enough to pay off your debt.`
+				accept
+		
+			label no
+			`	Alexandra nods her head sadly. "Are you sure, <first>? I've heard horror stories. First-time pilots who suffer a few setbacks, being chased down by loan officers... I couldn't bear the thought."`
+			choice
+				`	"On second thought, being debt-free sounds awesome. I'll do it."`
+					goto yes
+				`	"Sorry, it's just not for me. I'll do it the old-fashioned way."`
+			label endno
+			`	Alexandra sighs, then smiles. "Alright! Well, thank you for your time. Perhaps we'll bump into each other again!"`
+				decline
+	
+	on accept
+		set "liekedeeler agreement"
+		event "Liekedeeler Delivery 1" 8 10
+		log "People" "Alexandra Merrin" `Alexandra Merrin is a representative of a courier firm named 'Liekedeeler Enterprises.'`
+		log `Alexandra Merrin of Liekedeeler Enterprises has wired me enough money to pay my debts in exchange for some on-demand work in Republic space.`
+		payment 500000
+		
+#* * *  (Intro END)  * * *
+
+
+
+#* * * * * (Mission 1)  * * * * *
+event "Liekedeeler Delivery 1"
+
+mission "Liekedeeler Delivery [1]"
+	name "Liekedeeler delivery to <planet>"
+	description "Deliver <cargo> to <destination> for Liekedeeler Enterprises by <date>."
+	blocked `Alexandra contacts you just as you touch down on <origin>. "We have a job for you, <first>, but you're going to need <capacity>. Remember - we have a contract! Drop whatever you're doing and come back."`
+	deadline 2

```

Comment:
This makes the mission impossible to complete, because the destination planet is always between 6 and 10 jumps away, but you're only given two days to get it there.

Reply:
Unless you get really lucky with your jumpdrive routes.

Reply:
So, that's my bad - I had an old copy of the CreatingMissions wiki page (I do a lot of writing offline), which stated that `"a 'deadline 2' statement means offer the default deadline, plus two days"`. The current wiki clarifies this, but I never bothered to check it. That's why every mission has this bug; Alexandra wasn't lying about cutting costs by having the fastest couriers in the 'verse.